tree/dataframe/src/RDataFrame.cxx

```diff
@@ -456,6 +456,40 @@ executed at the moment they are called, but they are **lazy**, i.e. delayed unti
 accessed through the smart pointer. At that time, the event loop is triggered and *all* results are produced
 simultaneously.
 
+For yet another example of the difference between the correct and incorrect running of the event-loop, see the following
+two code snippets. We assume our ROOT file has branches a, b and c.
+
+The correct way - the dataset is only processed once.
+~~~{.cpp}
```

Comment:
Thanks for the example, could I ask you to also provide the Python equivalent and write it before the C++, or maybe just the Python version?